kubetest2-tf/deployer/deployer.go

```diff
@@ -50,7 +54,7 @@ type AnsibleInventory struct {
 }
 
 // Add additional Linux package dependencies here, used by checkDependencies()
-var dependencies = []string{"terraform", "ansible"}
+var dependencies = []string{"terraform", "ansible", "kubectl"}
```

Comment:
Essential for dumping cluster logs. Already present installed in the all-in-one image.